server.js

```diff
@@ -1,23 +1,81 @@
 const express = require("express");
 const cors = require("cors");
+const bp = require("body-parser");
+const fs = require("fs");
 
 const app = express();
+let data = JSON.parse(fs.readFileSync("data.json", "utf-8"));
+let maxID = Math.max(...data.map((c) => c.id));
+const messages = data;
 
 app.use(cors());
+app.use(bp.json());
+app.use(bp.urlencoded({ extended: true }));
 
-const welcomeMessage = {
-  id: 0,
-  from: "Bart",
-  text: "Welcome to CYF chat system!",
-};
-
-//This array is our "data store".
-//We will start with one message in the array.
-//Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+//Creates a message and post it to the database
+app.post("/messages", function (req, res) {
+  let userName = req.body.userName;
+  let userMessage = req.body.userMessage;
```

Comment:
These values don't change, so you can declare them as const instead.

The answers on this SO question give a good overview of why this is a good thing to do : https://stackoverflow.com/questions/136880/sell-me-on-const-correctness